test/functional/feature_loadblock.py

```diff
@@ -45,17 +45,17 @@ def run_test(self):
 
         self.log.info("Create linearization config file")
         with open(cfg_file, "a", encoding="utf-8") as cfg:
-            cfg.write("datadir={}\n".format(data_dir))
-            cfg.write("rpcuser={}\n".format(node_url.username))
-            cfg.write("rpcpassword={}\n".format(node_url.password))
-            cfg.write("port={}\n".format(node_url.port))
-            cfg.write("host={}\n".format(node_url.hostname))
-            cfg.write("output_file={}\n".format(bootstrap_file))
-            cfg.write("max_height=100\n")
+            cfg.write(f"datadir={data_dir}\n")
+            cfg.write(f"rpcuser={node_url.username}\n")
+            cfg.write(f"rpcpassword={node_url.password}\n")
+            cfg.write(f"port={node_url.port}\n")
+            cfg.write(f"host={node_url.hostname}\n")
+            cfg.write(f"output_file={bootstrap_file}\n")
+            cfg.write(f"max_height=100\n")
             cfg.write("netmagic=fcc1b7dc\n")
```

Comment:
22229: missed `f` here (to align with upstream even though `f` is not really needed here)